### DISCUSSION TITLE
Database table descriptions

### DISCUSSION THREAD

**Discussion by nipunarora** [photoprism/photoprism#2403] [General]
Are there any descriptions about the schema of the database tables? For instance I have about 40 or so albums in photoprism, but the `photo_album` and `album` tables in the database have significantly higher count. Where can I find out more about the tables, and their schema.

```
MariaDB [photoprism]> select count(*) from albums;
+----------+
| count(*) |
+----------+
|     1004 |
+----------+
1 row in set (0.002 sec)
```
```
MariaDB [photoprism]> select count(*) from photos_albums;
+----------+
| count(*) |
+----------+
|    11264 |
+----------+
1 row in set (0.008 sec)
```

**Comment by graciousgrey** [comment]
You can use a database tool of your choice to have a look at the database tables.

The photos_albums table shows which photos are part of what album. So there is one entry for each photo-album relation.

The album table shows all albums. 
Note in PhotoPrism there are multiple types of albums such as: album, folder, moment, month.
